_episodes_rmd/05-dplyr.Rmd

```diff
@@ -57,14 +51,14 @@ packages give you access to more functions. You need to install a package and
 then load it to be able to use it.
 
 ```{r, eval = FALSE, purl = FALSE}
-install.packages("dplyr") ## install
+install.packages("tidyverse") ## install
```

Comment:
I think for new learners it can be best to be explicit about packages instead of loading them all at once with `library(tidyverse)`. So maybe it would be better to use 

```r
library("dplyr")
library("readr")
```

instead?

Reply:
addressed in new commit of this pull request